pkg/openwsn/include/openwsn_uart.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2018 Hamburg University of Applied Sciences
+ *               2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_openwsn
+ * @{
+ *
+ * In RIOT, the first configured UART device is mapped to STDIO in most cases.
```

Comment:
Isn't this duplicate to the doc.txt?

Reply:
Yes, I wasn't sure if the best place would be the header or the pkg doc.

Reply:
I think that I would put it in the doc.txt and maybe point to it from here.

Reply:
done